tests/defences/trainer/test_adversarial_trainer_FBF.py

```diff
@@ -37,6 +37,9 @@ def _get_adv_trainer():
             trainer = AdversarialTrainerFBFPyTorch(classifier, eps=0.05)
         if framework == "scikitlearn":
             trainer = None
+        if framework == "huggingface":
```

Comment:
This case can now be combined with `pytorch` by changing line 35 to `if framework in ["pytorch", "huggingface"]:`